vsintegration/src/FSharp.ProjectSystem.FSharp/Project.fs

```diff
@@ -2464,9 +2464,9 @@ namespace rec Microsoft.VisualStudio.FSharp.ProjectSystem
 
                         let bodyString =
                             match location with
-                            | Above -> FSharpSR.FileCannotBePlacedBodyAbove
-                            | Below -> FSharpSR.FileCannotBePlacedBodyBelow
-                            |> FSharpSR.GetStringWithCR
+                            | Above -> FSharpSR.FileCannotBePlacedBodyAbove()
+                            | Below -> FSharpSR.FileCannotBePlacedBodyBelow()
+                            |> (fun s -> s.Replace("\n", Environment.NewLine))
```

Comment:
This is incorrect - this should be `@"\n"`.

Reply:
Good catch.